prov/shm/src/smr.h

```diff
@@ -448,5 +448,68 @@ static inline int smr_cma_loop(pid_t pid, struct iovec *local,
 	}
 }
 
+static inline struct smr_cmd_entry *
+smr_get_cmd(struct smr_region *smr, int64_t *pos)
+{
+	struct smr_cmd_entry *ce = NULL;
+	int ret;
+
+	ret = smr_cmd_queue_next(smr_cmd_queue(smr), &ce, pos);
+
+	if (ret == -FI_ENOENT)
+		return NULL;
+
+	return ce;
+}
+
+static inline void smr_queue_cmd(struct smr_cmd_entry *ce,
+				 int64_t pos)
+{
+	smr_cmd_queue_commit(ce, pos);
+}
```

Comment:
No need for a function that's a single line and calls directly through to another function.  It only complicates reading and code maintenance.